css/settings.css

```diff
@@ -32,10 +32,16 @@ input{
     display: none;
 }
 
+#embedUrl, #pubUrl {
+    vertical-align: text-bottom;
```

Comment:
This doesn't really seem to have hit the mark. It's not quite aligned, and it acts in two completely different ways depending on whether one is using firefox or chrome.

It seems like setting vertical align to `middle` for the URLs and the copy button seems to work as intended in both browsers.